starclass/utilities.py

```diff
@@ -95,3 +96,59 @@ def rms_timescale(lc, timescale=3600/86400):
 
 	# Compute robust RMS value (MAD scaled to RMS)
 	return mad_to_sigma * nanmedian(np.abs(flux_bin - nanmedian(flux_bin)))
+
+#--------------------------------------------------------------------------------------------------
+def get_periods(featdict, nfreqs, time, sorted=False, in_days=True):
```

Comment:
May I suggest to rename `sorted` to `ignore_harmonic` or something similar? It is not because the other way is not sorted... The flag is more whether it is ignoring the harmonic peaks (and only using the independent peaks) or not.

Reply:
Done!